pagefaultgames/pokerogue
# fix import eslint module not working in new flat eslint config

## What are the changes?

Fixes eslint rules not working for the eslint-plugin-import module

## Why am I doing these changes?



To fix eslint in preparation of cleanup of circular dependencies

## What did change?


Fixes the lines that conflict with the rules that now work, and fixes the eslint config

### Screenshots/Videos



## How to test the changes?




You can pull only the eslint config and package changes to see the eslint rules working on the broken files that were updated

## Checklist
